### ISSUE TITLE
404 links at https://ocm.software/docs/guides/getting-started-with-ocm/

### ISSUE THREAD

**Issue by sTb1337** [open-component-model/ocm-website#188]
**What happened**:
404 at https://ocm.software/docs/guides/getting-started-with-ocm for:
- `To do this, create a file named [.ocmconfig](https://ocm.software/docs/guides/getting-started-with-ocm/.ocmconfig) in your home directory with the following:`
- `The [component descriptor](https://ocm.software/docs/specification/elements/README.md#component-descriptor) is stored as a yaml file named component-descriptor.yaml. It describes the content of a component version.`

**Comment by morri-son** [comment]
@stb1337 I fixed everything, but the controller Api stuff.

@Skarlso, how do we create the API doc on the OCM website? Is this a copy/paste of some (never :-)) current state? What to reference to get the https://ocm.software/docs/controller-reference/api-reference/ocm-controller/#delivery.ocm.software/v1alpha1.ResourceReference fixed?

**Comment by Skarlso** [comment]
@morri-son I'll take a look at my notes. :D I don't remember if it was just generating the docs on the ocm-controller and the copying it over.

**Comment by Skarlso** [comment]
Yep, it's `make api-docs` on the controller side and then copy and paste it to the website.

**Comment by morri-son** [comment]
@Skarlso oki, thanks. Got it. Could you please also check my other comment? 

What to reference to get the https://ocm.software/docs/controller-reference/api-reference/ocm-controller/#delivery.ocm.software/v1alpha1.ResourceReference fixed?
